ns1/ipam.py

```diff
@@ -559,6 +559,28 @@ def success(result, *args):
 
         return self._rest.create(self.scopegroup_id, self.address_id, options = self.options, mac=self.mac, callback=success, errback=errback, **kwargs)
 
+    def update(self, options, callback=None, errback=None, parent=True, **kwargs):
+        """
+        Update reservation configuration. Pass a list of keywords and their values to
+        update. For the list of keywords available for address configuration, see :attr:`ns1.rest.ipam.Reservations.INT_FIELDS` and :attr:`ns1.rest.ipam.Reservations.PASSTHRU_FIELDS`
+        """
+        if not self.data:
+            raise ReservationException('Reservation not loaded')
+
+        def success(result, *args):
+            self.data = result
+            self.id = result['id']
+            self.address_id = result['address_id']
+            self.mac = result['mac']
+            self.options = result['options']
+            if callback:
+                return callback(self)
+            else:
+                return self
```

Comment:
do we have a convention preferring "useless else" in this codebase/py in general?

Reply:
Not sure, though these blocks I think are pretty consistent in this particular codebase